api/app/GraphQL/Mutations/ArchivePool.php

```diff
@@ -14,8 +14,9 @@ final class ArchivePool
      */
     public function __invoke($_, array $args)
     {
+        /** @var Pool $pool */
         $pool = Pool::find($args['id']);
-        if ($pool->getStatusAttribute() !== PoolStatus::CLOSED->name) {
+        if ($pool->status !== PoolStatus::CLOSED->name) {
```

Comment:
What's the difference between `status` and `getStatusAttribute`? I searched through the codebase and see `getStatusAttribute` used in other places still: should those also be replaced with `status`?

Reply:
Any method on a model that uses `get*Attribute` creates a magic accessor where you can use the `*` as a property.

Instead of adding the method to the model definition, I used the property directly since it is available. We may want to use the magic getter in the other places instead as well :thinking: I wonder why this is the only instance it flagged.